tests/import-object.test.ts

```diff
@@ -42,14 +46,24 @@ export const importObject : any = {
     //  We can then examine output to see what would have been printed in the
     //  console.
     assert_not_none: (arg: any) => assert_not_none(arg),
-    print: (arg: any) => print(Type.Num, arg),
-    print_num: (arg: number) => print(Type.Num, arg),
-    print_bool: (arg: number) => print(Type.Bool, arg),
-    print_none: (arg: number) => print(Type.None, arg),
-    abs: Math.abs,
-    min: Math.min,
-    max: Math.max,
-    pow: Math.pow,
+    print_num: (arg: number) => print(Type.Num, arg, importObject.libmemory.load),
+        print_bool: (arg: number) => print(Type.Bool, arg, null),
+        print_none: (arg: number) => print(Type.None, arg, null),
+        abs:  (arg: number) => builtin_bignum([arg], bigMath.abs, importObject.libmemory),
```

Comment:
The format below is nice – good use of `libmemory`, helpers to have clean code for these.